library/common/extensions/filters/http/socket_tag/filter.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto3";
+
+package envoymobile.extensions.filters.http.socket_tag;
+
+message SocketTag {
```

Comment:
let's comment somewhere (here?) about how socket tagging works and maybe why you want it (the bits you kindly already explained to me, but in writing =P)